articles/openshift/openshift-faq.md

```diff
@@ -85,3 +85,88 @@ Yes. You can use OSBA with Azure Red Hat OpenShift. See [Open Service Broker for
+
+We do not distinguish between the two types of maintenance. Our teams are available 24/7/365 and do not use traditional scheduled “out-of-hours” maintenance windows.
+
+## How will host operating system and OpenShift software be updated?
+
+The host operating system and OpenShift software is updated through our general upgrade and image build process.
+
+## What’s the process to reboot the updated node?
+
+This should be handled as a part of an upgrade.
+
+## Is data stored in etcd encrypted on ARO?
+
+It is not encrypted on the etcd level. The option to turn it on is currently unsupported. OpenShift supports this feature, but engineering efforts are required to make it on the road map. The data is encrypted at the disk level. Refer to [Encrypting Data at Datastore Layer](https://docs.openshift.com/container-platform/3.11/admin_guide/encrypting_data.html) for more information.
+
+## Can logs of underlying VMs be streamed out to a customer log analysis system?
+
+Syslog, docker logs, journal, and dmesg are handled by the managed service and are not exposed to customers.
+
+## How can a customer get access to metrics like CPU/memory at the node level to take action to scale, debug issues, etc. I cannot seem to run `kubectl top` on an ARO cluster.
+
+`kubectl top` is not available on Red Hat OpenShift. It requires a backing metrics source, either Heapster (deprecated) or metrics-server (incubating/alpha), neither of which are included in the OpenShift monitoring stack.
+
+## What is the default pod scheduler configuration for ARO?
+
+ARO uses the default scheduler that ships in OpenShift. There are a couple of additional mechanisms that are not supported in ARO. Refer to [default scheduler documentation](https://docs.openshift.com/container-platform/3.11/admin_guide/scheduling/scheduler.html#generic-scheduler) and [master scheduler documentation](https://github.com/openshift/openshift-azure/blob/master/pkg/startup/v6/data/master/etc/origin/master/scheduler.json) for more details.
+
+Advanced/Custom scheduling is currently unsupported. This could be something we plan for the future, but we do not have anything currently supported for ARO. Refer to [](https://docs.openshift.com/container-platform/3.11/admin_guide/scheduling/index.html) for more details.
+
+## If we scale up the deployment, how do Azure fault domains map into pod placement to ensure all pods for a service do not get knocked out by a failure in a single fault domain?
+
+There are by default five fault domains when using Azure VMScalesets. Each VM instance in a scale set will get placed into one of these fault domains. This ensures that applications deployed to the compute nodes in a cluster will get placed in separate fault domains.
+
+Refer to [Choosing the right number of fault domains for virtual machine scale set](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-manage-fault-domains) for more details.
```

Comment:
All links must not have locale info in them, please remove en-us

Reply:
Will remove.